eos-tech-support/eos-write-live-image

```diff
@@ -380,7 +380,15 @@ AUTORUN_INF
     fi
 fi
 
-"$EOS_WRITE_IMAGE" "$OS_IMAGE" "-" > "${DIR_IMAGES_ENDLESS}/endless.img"
+# attempt to avoid copy if input image is already compressed but fall back
+# gracefully if the image and tmpdir turn out to be different devices
+if [ "$OS_IMAGE" == "$OS_IMAGE_UNCOMPRESSED" ]; then
+    ln "$OS_IMAGE" "${DIR_IMAGES_ENDLESS}/endless.img" || true
```

Comment:
Might want to redirect `2>/dev/null` so it doesn't appear that something has gone wrong here.